spec/features/course_copying_spec.rb

```diff
@@ -53,6 +52,8 @@
     expect(page).not_to have_content 'Mark the holidays'
     click_button 'Save New Course'
 
+    sleep 0.5
+
     new_course = Course.last
```

Comment:
If you move this until after the `have_current_path` in the next line, I think you won't need the `sleep`. The problem is that `Course.last` happens immediately on the backend, before the results of the 'Save New Course' action have been saved to the DB, but once the path updates, it will already be saved.

Reply:
@ragesoss  Done!